x-pack/packages/kbn-elastic-assistant/impl/data_anonymization/settings/anonymization_settings_management/index.tsx

```diff
@@ -6,70 +6,127 @@
+    DEFAULT_PROMPTS, // Anonymization settings do not require prompts
+    false, // Anonymization settings do not require conversations
+    anonymizationFields ?? DEFAULT_ANONYMIZATION_FIELDS
+  );
+
+  const onCancelClick = useCallback(() => {
+    resetSettings();
+    setHasPendingChanges(false);
+  }, [resetSettings]);
+
+  const handleSave = useCallback(
+    async (param?: { callback?: () => void }) => {
+      await saveSettings();
+      toasts?.addSuccess({
+        iconType: 'check',
+        title: SETTINGS_UPDATED_TOAST_TITLE,
+      });
+      setHasPendingChanges(false);
+      param?.callback?.();
+    },
+    [saveSettings, toasts]
+  );
+
+  const onSaveButtonClicked = useCallback(() => {
+    handleSave();
+  }, [handleSave]);
+
+  const handleAnonymizationFieldsBulkActions = useCallback(
+    (value) => {
+      setHasPendingChanges(true);
+      setAnonymizationFieldsBulkActions(value);
+    },
+    [setAnonymizationFieldsBulkActions]
+  );
+
+  const handleUpdatedAnonymizationData = useCallback(
+    (value) => {
+      setHasPendingChanges(true);
+      setUpdatedAnonymizationData(value);
+    },
+    [setUpdatedAnonymizationData]
+  );
+
+  const onListUpdated = useAnonymizationListUpdate({
+    anonymizationFields: updatedAnonymizationData,
+    anonymizationFieldsBulkActions,
+    setAnonymizationFieldsBulkActions: handleAnonymizationFieldsBulkActions,
+    setUpdatedAnonymizationData: handleUpdatedAnonymizationData,
   });
   return (
-    <EuiPanel hasShadow={false} hasBorder paddingSize="l">
-      <EuiTitle size={'xs'}>
-        <h2>{i18n.SETTINGS_TITLE}</h2>
-      </EuiTitle>
-      <EuiSpacer size="s" />
-      <EuiText size={'xs'}>{i18n.SETTINGS_DESCRIPTION}</EuiText>
-
-      <EuiSpacer size="m" />
-
-      <EuiFlexGroup alignItems="center" data-test-subj="summary" gutterSize="none">
-        <Stats
-          isDataAnonymizable={true}
-          anonymizationFields={anonymizationFields.data}
-          titleSize="m"
-          gap={euiThemeVars.euiSizeS}
-        />
-      </EuiFlexGroup>
+    <>
+      <EuiPanel hasShadow={false} hasBorder paddingSize="l">
+        <EuiTitle size={'xs'}>
+          <h2>{i18n.SETTINGS_TITLE}</h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiText size={'xs'}>{i18n.SETTINGS_DESCRIPTION}</EuiText>
+
+        <EuiSpacer size="m" />
 
-      <EuiSpacer size="m" />
+        <EuiFlexGroup alignItems="center" data-test-subj="summary" gutterSize="none">
+          <Stats
+            isDataAnonymizable={true}
+            anonymizationFields={updatedAnonymizationData.data}
+            titleSize="m"
+            gap={euiThemeVars.euiSizeS}
+          />
+        </EuiFlexGroup>
 
-      <ContextEditor
-        anonymizationFields={anonymizationFields}
-        compressed={false}
-        onListUpdated={onListUpdated}
-        rawData={null}
-        pageSize={defaultPageSize}
+        <EuiSpacer size="m" />
+
+        <ContextEditor
```

Comment:
Overflow action icons are in a different orientation, color and have different padding between Anonymization and the other tables. 

@bojanasan, I know you had mentioned in the designs that you wanted the all the other tables to have their actions within the overflow popover for consistency with the Anonymization table (since it has four actions), but would you be willing to reconsider just showing the edit/delete directly on the table for the Conversations and System/Quick Prompts tables? This would resolve the other consistency issues with the Anonymization table actions, and TBH, the friction in having to do two clicks when managing a bunch of changes between these tabs is kinda wearing on me. Power user problems I suppose 😅  


Anonymization:
<img width="500" alt="image" src="https://github.com/elastic/kibana/assets/2946766/de57b889-4047-445c-96c4-24674a51c4ef">
Quick Prompts:
<img width="500" alt="image" src="https://github.com/elastic/kibana/assets/2946766/c57869bb-d642-4276-bf5f-695d9e92ce0f">

Reply:
@spong Sounds like the consistency is competing with ease of UX. Sure lets update the System/Quick prompts. I'll make changes to the mocks.